freqtrade/__init__.py

```diff
@@ -3,11 +3,13 @@
 
 if 'dev' in __version__:
     try:
+        import os
         import subprocess
+        freqtrade_basedir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Please use Pathlib - it provides the same functionalities - but in a safer method (safe as in - works cross-platform including windows).

I consider the os module as legacy when it comes to path handling - it had too many bugs in the past when it came to windows.

Reply:
done